code/modules/antagonists/traitor/contractor/datums/syndicate_contract.dm

```diff
@@ -518,14 +518,52 @@
 
 	QDEL_LIST(temp_objs)
 
-	// Chance for souvenir or bruises
+	// Injuries due to questioning and souvenirs
 	if(prob(RETURN_SOUVENIR_CHANCE))
 		to_chat(M, "<span class='notice'>Your captors left you a souvenir for your troubles!</span>")
 		var/obj/item/souvenir = pick(souvenirs)
 		new souvenir(closet)
-	else if(prob(RETURN_BRUISE_CHANCE) && M.health >= 50)
-		to_chat(M, "<span class='warning'>You were roughed up a little by your captors before being sent back!</span>")
-		M.adjustBruteLoss(RETURN_BRUISE_DAMAGE)
+	if(prob(RETURN_INJURY_CHANCE) && M.health >= 50)
+		var/obj/item/organ/external/injury_target
+		var/nolimb = FALSE
```

Comment:
```suggestion
		var/do_damage = FALSE
```
Or something the like. This makes it more clear what setting the var will do later in the chain